holoviews/core/util.py

```diff
@@ -445,6 +445,10 @@ def argspec(callable_obj):
         arglen = len(callable_obj.args)
         spec = inspect.getfullargspec(callable_obj.func)
```

Comment:
I think we should unpack the partial before this line and maybe at the start of the elif statement.

Reply:
Maybe but I'm not really sure.

In fact the following also works and yields the same `args` (both for simple partial method and nested partial methods):
```python
elif isinstance(callable_obj, partial): # partials
    arglen = len(callable_obj.args)
    while isinstance(callable_obj.func, partial): # Working with the inner-most partial from here
        callable_obj = callable_obj.func
    spec = inspect.getfullargspec(callable_obj.func)
    args = [arg for arg in spec.args[arglen:] if arg not in callable_obj.keywords]
    if inspect.ismethod(callable_obj.func):
        args = args[1:]
```

Reply:
Well in fact we don't even have to loop because the outer partial's `func` object is already the original method. It looks like nesting partials is akin to merging the keywords supplied at the different levels, so the result is not really nested.

Therefore I would argue the change required is really minimal and only involves checking whether `callable_obj.func` is a method or not.

Reply:
Great to hear.

Reply:
And don't worry about unrelated failing tests, I'm working on them in https://github.com/holoviz/holoviews/pull/6078